config.json

```diff
@@ -75,6 +75,19 @@
         ],
         "difficulty": 2
       },
+      {
+        "slug": "reverse-string",
+        "name": "Reverse String",
+        "uuid": "5d28be8e-395e-4e2a-aab6-c5777bcb0668",
+        "practices": [
+          "stack-effect",
+          "strings"
+        ],
+        "prerequisites": [
+          "stack-effect"
```

Comment:
Shouldn't `strings` also be a prerequisite here? I think you have to know about strings before being able to reverse a string, right?

Reply:
Hmm maybe I'm fuzzy on the idea of what "prerequisites" is supposed to mean - I was thinking someone could learn strings while doing this?

I'll add it in, I'm probably expecting too much

Reply:
Adding and rebasing

Reply:
@hiljusti It sounds like you got prerequisites and practices the other way around :) Prequisites are the things the student _must_ be familiar with in order for the student to be able to solve the exercises. Practices are the things the exercise can be used for to practice certain concepts on. So e.g. an exercise could have `strings` as a prerequisites (e.g. as the input or output parameter is a string), but have `recursion` as a a practices concept as the student _could_ solve the exercise with recursion.

See https://github.com/exercism/docs/blob/main/building/tracks/config-json.md#practice-exercises